contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+3. Studies investigating churn rates by industry use different methodologies and data sets. They often come to conflicting conclusions.
+
+4. Companies who self report churn rate are liable to, erm... lie. After all, who wants to admit they have a churn problem? They might also define churn in their own, special way.
+
+5. What's "good" for a mature business is very different to what's good for a pre-product-market fit, early-stage startup.
+
+With all that said, and having read countless studies, meta-analyses, and finger-in-the-air opinions on the topic, here are my conclusions:
+
+- **Monthly churn of 0.5% or less** is outstanding for any company, but an unrealistic expectation for B2C products. It's equivalent to an annual churn rate of 5.84% – i.e. a company with 1,000 customers would lose 58 over 12 months.
+
+- **Monthly churn of 1.5 to 2%** is still very good, especially for mid-market B2B companies that don't sell to enterprises. It's equivalent to an annual churn rate of 16.59% – i.e. a company with 1,000 customers would lose 166 over 12 months.
+
+- **Monthly churn of 3.5%** or 34.8% per year is pretty typical for B2B SaaS products with less than $250 average revenue per customer (ARPC). The higher your ARPC, the lower your churn rate ought to be. 
+
+- While an **annual churn of 45%** (aka 5% per month) sounds terrifying, it's not unheard of among successful mid-market B2B SaaS companies. In 2016, Buffer, a successful SaaS company with over $1 million in monthly recurring revenue at the time, reported annual churn of 46%. Multiple studies show a 5% monthly churn rate is common among smaller B2B SaaS companies.
+
+- **Monthly churn above 5%** is to be expected for any early-stage company B2B SaaS, and a good benchmark for B2C subscription products. When you're early-stage, hitting an absolute number is less important than seeing your churn rate improving.
+
+See [more on churn rate benchmarks](#more-on-churn-rate-benchmarks) in the appendix for some of the sources I used to reach these conclusions.
+
+### Problems with churn rate and alternative methods
+
+Other articles you read about churn rate may introduce alternative ways to measure it lifted directly from [this 2011 article](https://shopify.engineering/defining-churn-rate-no-really-this-actually-requires-an-entire-blog-post) by a data scientist at Shopify.
+
+It's a dense affair that highlights some interesting quirks about churn rate, which I'll quickly summarize:
+
+- Extreme growth can lead to distorted churn figures – such as reporting lower churn when user behavior hasn't actually changed. High-growth startups should be wary as a result.
+
+- Because churn rate measures churn among **existing customers**, it can hide churn problems among new, incoming customers.
+
+The Shopify article explores several complex alternative churn formulas to address these problems. It's a laudable objective, but in my view **totally unnecessary**.
```

Comment:
Not sure you have made an actual argument here that the Shopify article is wrong? I'd consider skipping the whole 'problems with alternative methods' section as you don't really go deep enough here - to do this properly, you'd have to explain each alternate method, then rebut it, which takes us into academic paper territory...

Reply:
Agree, if someone is at this point I think they are bought into churn rate as a metric. I'd never heard of that Shopify article and it is 12 years old. I think this section could derail a lot of readers.